data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -200,498 +186,105 @@
-    parser.add_argument('-d',
-                        '--data_stage',
-                        required=True,
-                        dest='data_stage',
-                        action='store',
-                        type=stage,
-                        choices=list(
-                            [s for s in stage if s is not stage.UNSPECIFIED]),
-                        help='Specify the dataset')
-    parser.add_argument('-s', action='store_true', help='Send logs to console')
-    parser.add_argument('-p',
-                        '--project-id',
-                        dest='project_id',
-                        action='store',
-                        help=('Project identifier for cleaning'))
-    parser.add_argument('-n',
-                        '--dataset-id',
-                        dest='dataset_id',
-                        action='store',
-                        help=('Dataset identifier for cleaning'))
-    deid_group = parser.add_argument_group(
-        'post_deid',
-        'Arguments related to the deid mapping table location that should be used for post-deid cleaning rules.'
-    )
-    deid_group.add_argument(
-        '--mapping-dataset',
-        dest='mapping_dataset',
-        action='store',
-        help=('Supply dataset name for dataset containing the _deid_map table '
-              'to reference.'))
-    deid_group.add_argument(
-        '--mapping-table',
-        dest='mapping_tablename',
-        action='store',
-        help=(
-            'Define the mapping table name.  Default historically is _deid_map.'
-        ))
-    args = parser.parse_args()
-    clean_engine.add_console_logging(args.s)
-    if args.data_stage == stage.EHR:
-        clean_ehr_dataset()
-    elif args.data_stage == stage.UNIONED:
-        clean_unioned_ehr_dataset()
-    elif args.data_stage == stage.RDR:
-        clean_rdr_dataset()
-    elif args.data_stage == stage.COMBINED:
-        clean_combined_dataset()
-    elif args.data_stage == stage.DEID_BASE:
-        clean_combined_de_identified_dataset()
-    elif args.data_stage == stage.DEID_CLEAN:
-        clean_combined_de_identified_clean_dataset()
-    elif args.data_stage == stage.FITBIT:
-        if args.project_id and args.dataset_id and args.mapping_dataset and args.mapping_tablename:
-            clean_fitbit_dataset(args.project_id, args.dataset_id,
-                                 args.mapping_dataset, args.mapping_tablename)
-        else:
-            raise RuntimeError(
-                "Please run `python cdr_cleaner/clean_cdr.py -h` to list help "
-                "commands.  One or more of project-id, dataset-id, "
-                "mapping-dataset, or mapping-table are missing.  These are "
-                "required to clean fitbit data.")
+    args, kwargs = fetch_args_kwargs()
+
+    rules = DATA_STAGE_RULES_MAPPING[args.data_stage]
+    for rule in rules:
+        clazz = rule[0]
+        kwargs.update({'sandbox_dataset_id': None})
```

Comment:
I think this code might smell better if `sandbox_dataset_id` were more like `project_id` and `dataset_id`.